### ISSUE TITLE
Issue with queries having negative integers

### ISSUE THREAD

**Issue by satyammittal** [mozilla/moz-sql-parser#8]
If query given is :  select B,C from table1 where A=-900 or B=100
Then current parsed query is 
{u'from': u'table1', u'where': {u'eq': [u'A', {u'neg': {u'or': [900, {u'eq': [u'B', 100]}]}}]}, u'select': [{u'value
': u'B'}, {u'value': u'C'}]} 

But I think, It would be much easier if solution would be
{u'from': u'table1', u'where': {u'or': [{u'eq': [u'A', -900]}, {u'eq': [u'B', 100]}]}, u'select': [{u'value': u'B'}, {u'value': u'C'}]}

**Comment by klahnakoski** [comment]
Good catch!  Thank you.  Your example exposed not just bad negative integers, but also bad order of operations.  I added tests for each, and fixed.  Let's see if this passes

https://github.com/mozilla/moz-sql-parser/commit/63b9a8921dded55c9ec99c8cae90ee127808409e

**Comment by satyammittal** [comment]
Great!!
Thanks
